src/main/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringcreateanorderapi/listener/CourtHearingEventListener.kt

```diff
@@ -16,24 +17,39 @@ class CourtHearingEventListener(
   private val eventHandler: HearingEventHandler,
   private val deadLetterQueueService: DeadLetterQueueService,
   private val objectMapper: ObjectMapper,
+  private val eventService: EventService,
 ) {
 
   @SqsListener("courthearingeventqueue", factory = "hmppsQueueContainerFactoryProxy")
   fun onDomainEvent(rawMessage: String) {
+    val startTimeInMs = System.currentTimeMillis()
     try {
       val eventMessage: HmppsSqsEventMessage = objectMapper.readValue(rawMessage)
       val courtHearing: HearingEvent = objectMapper.readValue(eventMessage.message)
 
       // Process message if contains EM details, else ignore
       if (courtHearing.hearing.isHearingContainsEM()) {
-        val result = eventHandler.handleHearingEvent(courtHearing)
-        if (result.any()) {
-          deadLetterQueueService.sentEvent(rawMessage, result.joinToString(","))
+        val handlingErrors = eventHandler.handleHearingEvent(courtHearing)
+        if (handlingErrors.any()) {
+          deadLetterQueueService.sentEvent(rawMessage, handlingErrors.joinToString(","))
         }
+      } else {
+        val containsEmLabel = rawMessage.contains("Notification of electronic monitoring order")
```

Comment:
Added check if payload contains `"Notification of electronic monitoring order"`, as usually this is an indicator it has em order. This is used to identify how many we are potentially missing mapping.